.devcontainer/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM debian:latest
+
+RUN apt-get update \
+ && apt-get install -y \
+        gawk \
+        perl \
+        sed \
+        git \
+        python3 \
+        python3-pip \
+        pandoc \
+        biber \
+        latexmk \
+        texlive \
+        texlive-science \
+        texlive-fonts-extra \
+        texlive-plain-generic \
+        texlive-bibtex-extra \
+ && apt-get clean
+
+# Install Python packages, after removing a file that breaks pip.
+RUN rm /usr/lib/python3.11/EXTERNALLY-MANAGED
+RUN pip install rst2html5 \
+ && pip install docutils
```

Comment:
```suggestion
 && pip install docutils==0.19
```
This avoids unnecessary churn when different people regenerate the HTML. Version 0.19 is not the most recent but matches the version in Debian bookworm.